src/daemon/rpmostreed-os.cxx

```diff
@@ -339,28 +339,23 @@ get_deployments_rpm_diff (const char *arg_deployid0, const char *arg_deployid1,
 {
   g_autoptr (GCancellable) cancellable = NULL;
   RpmostreedSysroot *global_sysroot;
-  glnx_unref_object OstreeDeployment *deployment0 = NULL;
-  glnx_unref_object OstreeDeployment *deployment1 = NULL;
   OstreeSysroot *ot_sysroot = NULL;
   OstreeRepo *ot_repo = NULL;
   g_autoptr (GVariant) value = NULL;
-  const gchar *ref0;
-  const gchar *ref1;
 
   global_sysroot = rpmostreed_sysroot_get ();
 
   ot_sysroot = rpmostreed_sysroot_get_root (global_sysroot);
   ot_repo = rpmostreed_sysroot_get_repo (global_sysroot);
 
-  if (!rpmostreed_deployment_get_for_id (ot_sysroot, arg_deployid0, &deployment0, error))
-    return FALSE;
-  ref0 = ostree_deployment_get_csum (deployment0);
+  std::string deploy_id0 (arg_deployid0 ?: "");
```

Comment:
I think this can be:
`rust::Str deploy_id0 (arg_deployid0 ?: "");`
to avoid copying the string.  We can efficiently take a C string and wrap it as a Rust `&str`, just not the other way around.